app/lib/account/agent.dart

```diff
@@ -33,7 +38,8 @@ bool isValidUserId(String userId) {
 
 /// Whether the [agent] is a valid-looking identifier.
 bool isKnownServiceAgent(String agent) {
```

Comment:
What is a "KnownServiceAgent" I don't think I understand the concept anymore.

Should it instead be called "ServiceAgent" or `AutomatedServiceAgent` or ?

All this method really says is: "This is not a user".

Reply:
This method is effectively only used in integrity checks: https://github.com/dart-lang/pub-dev/blob/master/app/lib/shared/integrity.dart#L692-L696

I think this is slightly more than "This is not a user", it also verifies that the `agentId` string on the audit log record is valid. We could remove it if you think it is just better to use `isValidUserId` in the above use in the integrity check, but I'd rather expand on the documentation instead.